server/src/main/java/org/candlepin/resteasy/filter/AuthenticationFilter.java

```diff
@@ -91,6 +96,11 @@ public void setHttpServletRequest(HttpServletRequest request) {
     }
 
     public void setupAuthStrategies() {
+        // use keycloak authentication
+        if (config.getBoolean(ConfigProperties.KEYCLOAK_AUTHENTICATION) &&
```

Comment:
Remember to default to something reasonable (false) in these cases so we can avoid crashing unnecessarily.

Reply:
Willl work on the suggestions!

Reply:
@kahowell Needs the default value set to avoid an exception.

Reply:
We added it as a default in https://github.com/candlepin/candlepin/blob/009b0ff1087d4ec2fb19b50f8d4f41adf8acc20d/server/src/test/java/org/candlepin/resteasy/filter/AuthenticationFilterTest.java#L120 ; that should be sufficient, no?

If not, I'll make another commit to make sure everywhere we use it has the default.

Reply:
See updated link if you already read.